easybuild/easyconfigs/p/PGI/PGI-16.1-GCC-4.9.2-2.25.eb

```diff
@@ -0,0 +1,26 @@
+name = 'PGI'
+version = '16.1'
+
+homepage = 'http://www.pgroup.com/'
+description = "C, C++ and Fortran compilers from The Portland Group - PGI"
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+sources = ['pgicdk-%(version_major)s%(version_minor)s.tar.gz']
```

Comment:
@valtandor I guess the answer to your question depends on how the MVAPICH2 and MPICH libraries are provided... It's probably doesn't hurt to have those around, since the MPI library that is a part of the toolchain should win?